source/technical/daypass.rst

```diff
@@ -0,0 +1,91 @@
+.. _daypass:
+
+=======
+Daypass
+=======
+Normally, only Chameleon users with active allocations are able to launch and
+view Trovi artifacts. To allow anyone to launch an artifact, we also provide
+daypass. This allows for a non-Chameleon user to have access to Chameleon
+for a limited amount of time, using a small, separate allocation.
```

Comment:
Maybe emphasize it's linked to your main allocation (it's a "sub-allocation"?) and it's only for reproducibility purposes (i.e., not new work or long-term use.)

And perhaps a sentence: "This can be enabled on an artifact-by-artifact basis in Trovi."

Reply:
Can you help me understand the reason for why a user should know it is a sub-allocation? The only relevant detail that I am thinking of is that we use the main allocation as a source of pre-existing trust in the user. I am trying to prevent the issue where the user thinks this is using SUs from their main allocation, since these really are separate.

Reply:
OK, that makes sense! I was thinking that maybe the word allocation by itself makes it seem like it's a bit "unmoored" and the user might wonder, how does this work. But it also muddies the text to add more details there and I am not sure it is necessary. So, let's keep it as you have it then :)

Reply:
Later, I added "After these items are saved, a reproducibility allocation request is
automatically made under your PI's name". This should make it clearer.